crates/protocol/interop/src/access_list.rs

```diff
@@ -0,0 +1,27 @@
+use crate::CROSS_L2_INBOX_ADDRESS;
+use alloc::vec::Vec;
+use alloy_eips::eip2930::AccessListItem;
+use alloy_primitives::B256;
+
+/// Parses [`AccessListItem`]s to inbox entries.
+///
+/// See [`parse_access_list_item_to_inbox_entries`] for more details. Return flattened iterator with
+/// all inbox entries.
+pub fn parse_access_list_items_to_inbox_entries<'a>(
+    access_list_items: impl Iterator<Item = &'a AccessListItem>,
+) -> impl Iterator<Item = B256> {
+    access_list_items.filter_map(parse_access_list_item_to_inbox_entries).flatten()
+}
+
+/// Parse [`AccessListItem`] to inbox entries, if any.
+///
+/// TODO: add url to spec once  [pr](https://github.com/ethereum-optimism/specs/pull/612) is merged
+/// Max 3 inbox entries can exist per [`AccessListItem`] that points to `CROSS_L2_INBOX_ADDRESS`.
+/// Returns `Vec::new()` if [`AccessListItem`] address doesn't point to `CROSS_L2_INBOX_ADDRESS`.
```

Comment:
```suggestion
/// Max 3 inbox entries can exist per [`AccessListItem`] that points to [`CROSS_L2_INBOX_ADDRESS`].
/// Returns `Vec::new()` if [`AccessListItem`] address doesn't point to [`CROSS_L2_INBOX_ADDRESS`].
```
link where possible

Reply:
Thanks!
I'm a little confused on where to use [Smthng] and where to use [`Smthng`]

Reply:
back ticks do code formatting, so whenever smthg is code, use backticks. see how it's done by rust std lib https://doc.rust-lang.org/std/

Reply:
[some software engineer topic] <- that's just a link 
[`MyTrait`] <- that's a link with code formatting